as_pacmod.dbc

```diff
@@ -479,6 +482,7 @@ BO_ 556 STEERING_RPT: 8 PACMOD
  SG_ MANUAL_INPUT : 15|16@0- (0.001,0) [-32.768|32.767] "rad"  CUSTOMER_ECU
  SG_ COMMANDED_VALUE : 31|16@0- (0.001,0) [-32.768|32.767] "rad"  CUSTOMER_ECU
  SG_ OUTPUT_VALUE : 47|16@0- (0.001,0) [-32.768|32.767] "rad"  CUSTOMER_ECU
+ SG_ COMMAND_FAULT : 7|1@0+ (1,0) [0|1] ""  CUSTOMER_ECU
```

Comment:
I thought we were going to call these COMMAND_TIMEOUTs? Did we not make this change? Reasoning: A FAULT indicates a problem with a given system. The system itself is not having a problem in this case, it's just reached a timeout value which caused it to disable. I don't believe we made a determination as to whether this would disable all other systems and the convention so far has been that any fault on any system would keep other systems from enabling.

Reply:
I was initially opposed to this since none of the other fault types refer to the physical phenomena that caused it.  After discussion with Josh, I agree that this case is different since it is coming from outside of the PACMod System.  If everybody is okay with his, I will make the change.

Reply:
Works for me

Reply:
Though I agree with Andy's initial position, I'm fine with the change.